dispatcher/MongoConnect.py

```diff
@@ -418,15 +481,14 @@ def set_stop_time(self, number, detectors, force):
             self.log.error(f"Database having a moment, hope this doesn't crash. {type(e)}, {e}")
         return
 
-    def get_ack_time(self, detector, command):
+    def get_cc_ack_time(self, detector, command):
         '''
         Finds the time when specified detector's crate controller ack'd the specified command
         '''
-        cc = list(self.latest_status[detector]['controller'].keys())[0]
-        query = {f'acknowledged.{cc}': {'$ne': 0},
-                 '_id': self.command_oid[detector][command]}
+        query = {'_id': self.command_oid[detector][command]}
         doc = self.collections['outgoing_commands'].find_one(query)
-        if doc is not None and not isinstance(doc['acknowledged'][cc], int):
+        cc = list(self.latest_status[detector]['controller'].keys())[0]
```

Comment:
Am I missing something or is this a big problem for linked mode, if we have more than one CC, it will not acknowledge due to the `[0]`.

Reply:
Why?